app/src/main/java/com/peterlaurence/trekme/core/map/data/dao/MapSeekerDaoImpl.kt

```diff
@@ -69,8 +75,8 @@ class MapSeekerDaoImpl(
         }
 
         /* Find the first image */
-        val imageFile = findFirstImage(file, 0, 5) ?: throw
-        FileBasedMapParseException(Issue.NO_IMAGES)
+        val imageFile =
+            findFirstImage(file, 0, 5) ?: throw FileBasedMapParseException(Issue.NO_IMAGES)
```

Comment:
Maybe introduce a file private constant here since it`s called only in one place?

Reply:
We can leave it like that.